examples/pytorch/ogb/ngnn_seal/main.py

```diff
@@ -508,6 +501,12 @@ def print_log(*x, sep="\n", end="\n", mode="a"):
 
     if 0 < args.sortpool_k <= 1:  # Transform percentile to number.
         if args.dataset.startswith("ogbl-citation"):
+        # For this dataset, subgraphs extracted around positive edges are
```

Comment:
indent by 4 more, the code which explains the logic below should align with the code.